### PR TITLE
Implement issue #20

### PR DESCRIPTION
**Pull request by Zetta1Reid0**

### CODE REVIEW

**Comment by CleoMenezesJr** [comment]
Thanks for the pull request. Later, when I have time, I'll be reviewing.

File: autoActivities.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ var AutoActivities = GObject.registerClass(
 		      if (this._settings.get_boolean("show-apps"))
 			      Main.overview.showApps();
 		      else
-			      Main.overview.secondWorkspaceIndexhow();
+			      Main.overview.show();
```
Comment (CleoMenezesJr) [inline autoActivities.js:179]: Could you please inform better which buggy popup this would be and why the change?

File: prefs.ui [hunk 2]
```diff
@@ -2,6 +2,18 @@
 <interface domain="auto-activities">
   <object class="AdwPreferencesGroup" id="MainWidget">
     <property name="title" translatable="yes">Preferences</property>
+    <child>
```
Comment (CleoMenezesJr) [inline prefs.ui:5]: Could you please put `AdwActionRow` after _Always Hide Overview on New Window_?

File: prefs.ui [hunk 3]
```diff
@@ -2,6 +2,18 @@
...
+      <object class="AdwActionRow">
+        <property name="title" translatable="yes">Show Apps</property>
```
Comment (CleoMenezesJr) [inline prefs.ui:7]: It would be interesting to add a subtitle with something like "Show apps grid" or something like that. I think saying "Show Apps" might be a bit confusing.

**Comment by CleoMenezesJr** [comment]
@Zetta1Reid0 I just tested it, and it works quite well. Thanks a lot for the PR, sorting out the reviews, it would be ready to be merged.

**Comment by CleoMenezesJr** [comment]
Could you do a rebase with more semantic commits? Something like `feat: add option to show AppGrid`

**Comment by Zetta1Reid0** [comment]
Ok I did your suggestions . But instead of rebase i used soft reset to undo my commits then commited changes again.

**Comment by CleoMenezesJr** [comment]
Thanks, after some cosmetic tweaks this change will be in the next version.
